src/panels/lovelace/cards/hui-weather-forecast-card.ts

```diff
@@ -283,7 +283,7 @@ class HuiWeatherForecastCard extends LitElement implements LovelaceCard {
                         ${dayNight
                           ? html`
                               ${new Date(item.datetime).toLocaleDateString(
```

Comment:
We should use `formatDate` here

Reply:
`formatDate` does not support the options needed here to format the date as just the short weekday. I can submit a separate PR to add that capability and go through the codebase to use it where appropriate after this PR is merged.

Reply:
Could be the next `locale` option, date/time format 😅

Reply:
Yes the thought has crossed my mind 😳